lib/util/glob-util.js

```diff
@@ -41,6 +41,8 @@ debug = debug("eslint:glob-util");
 function processPath(extensions) {
     var suffix = "/**";
 
+    // This is a private function and will always be called with extensions.
+    /* istanbul ignore else */
```

Comment:
As the comment says the function is always called with a valid value for `extensions` so I would remove the if statement.